tgui/packages/tgui/components/TextArea.js

```diff
@@ -1,36 +1,62 @@
 /**
  * @file
  * @copyright 2020 Aleksej Komarov
- * @author Warlockd
+ * @author Original Aleksej Komarov
+ * @author Changes Warlockd (https://github.com/warlockd)
  * @license MIT
  */
 
-import { classes } from 'common/react';
+
+import { classes, canRender } from 'common/react';
 import { Component, createRef } from 'inferno';
 import { Box } from './Box';
-import { toInputValue } from './Input';
```

Comment:
Might want to change toInputValue from Input, but I'm not confortable enough with tgcode to do it, might break something

Reply:
If it's the same code, just do it.